types/zen-observable/index.d.ts

```diff
@@ -59,4 +59,4 @@ declare class Observable<T> {
 declare namespace Observable {
 }
 
-export = Observable;
+export default Observable;
```

Comment:
The root should continue to be an `export =`, since it's actually a `module.exports =` package. Users can use `esModuleInterop` if necessary. See https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default